docs/index.html

```diff
@@ -426,7 +458,7 @@ <h1 id="api-commonformats">Common Formats</h1>
 // $dt->toAtomString() is the same as $dt->format(DateTime::ATOM);
 echo $dt->toAtomString();      // 1975-12-25T14:15:16-05:00
 echo $dt->toCookieString();    // Thursday, 25-Dec-1975 14:15:16 EST
-echo $dt->toIso8601String();   // 1975-12-25T14:15:16-0500
+echo $dt->toIso8601String();   // 1975-12-25T14:15:16-05:00
```

Comment:
Before that, we should investigate: https://github.com/briannesbitt/Carbon/pull/977